gwdatafind/http.py

```diff
@@ -89,13 +100,19 @@ def _request_response(self, method, url, **kwargs):
         RuntimeError
             if query is unsuccessful
         """
-        self.request(method, url, **kwargs)
+        if self.bearer_token:
+            auth_header = {'Authorization' :
+                           'Bearer {}'.format(self.bearer_token)}
+            self.request(method, url, header=auth_header, **kwargs)
+        else:
+            self.request(method, url, **kwargs)
```

Comment:
This probably needs to handle the case where `header` is already given, and not override an existing `Authorization` header:

```suggestion
        if self.bearer_token:
            # add the bearer token as an Authorization header
            kwargs.setdefault("header", {}).setdefault(
                "Authorization",
                "Bearer {}".format(self.bearer_token),
            )
        self.request(method, url, **kwargs)
```